service/src/main/resources/api/openapi.yml

```diff
@@ -62,6 +65,8 @@ paths:
 
         ProfilePath must not be empty, as it is an error to attempt to overwrite the entire
         profile object. On success, the value to set will be returned.
+
+        Supplying an email will target that user's profile (requires Admin).
```

Comment:
If left blank, will target the current user.

Reply:
If userEmail is set, requester must be SAM admin, and userEmail's profile is returned. If userEmail is not set, requester's profile is returned.